argsParser.js

```diff
@@ -24,6 +24,12 @@ export default class ArgsParser {
         describe: 'Maximum length of the secret',
         default: Constants.DEFAULT_MAX_SECRET_LENGTH
       })
+      .option('d', {
+        alias: 'dictionary',
+        type: 'string',
+        describe: 'Password file to use instead of the brute force',
+        default: null
+      })
```

Comment:
we could also add a

```js
.conflicts('d', 'a')
```

To basically error if a user is specifying both the alphabet and the dictionary (which I believe should be mutually exclusive)

Reply:
to be tested btw, i have only looked at the tdoc and discovered that this is supported in some version of Yargs!